### PR TITLE
Few minor tweaks

### PR DESCRIPTION
**Pull request by JosephWoodward**

### CODE REVIEW

File: src/Magneto/Dispatcher.cs [hunk 1]
```diff
@@ -20,68 +20,42 @@ public Dispatcher(IServiceProvider serviceProvider, IInvoker invoker)
 		protected IInvoker Invoker { get; }
 
 		protected virtual TContext GetContext<TContext>()
-		{
-			return (TContext)ServiceProvider.GetService(typeof(TContext));
-		}
+			=> (TContext)ServiceProvider.GetService(typeof(TContext));
```
Comment (shaynevanasperen) [inline src/Magneto/Dispatcher.cs]: The code style convention in this project is to have the `=>` operator appear on the same line as the method signature.

Comment (slang25) [reply src/Magneto/Dispatcher.cs]: Don't trust him @JosephWoodward, he prefers tabs over spaces https://www.hanselman.com/blog/TabsVsSpacesAPeacefulResolutionWithEditorConfigInVisualStudioPlusNETExtensions.aspx#b889a115-83cc-4377-9746-533b01f59a3b

Comment (JosephWoodward) [reply src/Magneto/Dispatcher.cs]: lol, I prefer tabs over spaces :) That's been sorted @shaynevanasperen. As an aside, why do you prefer the lambda operator at the end? at the beginning seems far more readable as you can instantly see it's an expression bodied function call without having to cast your eye to the far right of the method signature?

Comment (slang25) [reply src/Magneto/Dispatcher.cs]: I do share the same opinion, although I am cautious to do so given you prefer tabs to spaces, how can I trust your judgement?

**Comment by JosephWoodward** [comment]
This has been updated and awaiting review @shaynevanasperen

**Comment by JosephWoodward** [comment]
I have a feeling this is never going to get merged 🤔

File: src/Magneto/Dispatcher.cs (hunk 1 above)
Comment (shaynevanasperen) [reply src/Magneto/Dispatcher.cs]: @JosephWoodward As explained to you in person, my preference for havin the lambda operator at the end of the line was to match the style used for when passing in-line lambdas as arguments to method calls. I see your point though and may choose to switch to having them on a new line in a later refactoring.

**Comment by slang25** [comment]
omg
It happened!!!!

**Comment by JosephWoodward** [comment]
alt text

**Pull request merged**